massa-network/src/network_worker.rs

```diff
@@ -59,6 +59,8 @@ pub enum NetworkCommand {
     Ban(NodeId),
     BanIp(Vec<IpAddr>),
     Unban(Vec<IpAddr>),
+    Whitelist(Vec<IpAddr>),
+    RemoveFromWhitelist(Vec<IpAddr>),
```

Comment:
Hum, yes because `RemoveFromWhitelist != BlackList` …

Reply:
Indeed we do not have a blacklist